doc/adoc/user_guide.adoc

```diff
@@ -0,0 +1,102 @@
+
+The distribution migration system provides the live image and a component that modifies the bootloader configuration such that on the next boot the system will boot into the upgrade image. Once booted into the upgrade live image the following chain of services will be executed:
+
+- detect the system to be migrated
+- mount the necessary file systems
+- setup the network to match the network configuration of the system to be migrated
+- prepare the package manager for the migration task
+- migrate the system
+- update the boot loader configuration
+- unmount all mounted file systems
+- reboot
+
+In case an error occurs prior to the start of the upgrade the system will be reverted to its original state.
+
+== Requirements
+Systems that are to be upgraded need to be registered. "Pay as you go"-instances in the Public Cloud are automatically registered to the SUSE operated update infrastructure. All other systems must be connected to the SUSE Customer Center (SCC), a Subscription Management Tool (SMT), or a Repository Management tool (RMT) server. For systems managed via SUSE Manager use the upgrade path provided by SUSE Manager. 
+
+== Recommendation
+During the upgrade it is only possible to log in via SSH key-based login. If your system is not configured for it, it is recommended that at least one of the users on the system has a `~/.ssh/authorized_keys` file with a private key accessible by the person executing the system upgrade.
+
+== Installation
+
+The distribution migration system is available from the Public Cloud module. Therefore this module has to be enabled on the system prior to upgrade. For running on-demand instances this module is already enabled.
+
+For data center customers it is recommended to continue to use the documented offline distribution upgrade using the next distribution version installation media.
+
+To install the distribution migration system run:
+
+[listing]
+tux > sudo zypper in SLE15-Migration suse-migration-sle15-activation
+
+== Run the Migration
+
+The `suse_migration_activation package` configures the boot loader to boot into the live image which will start the upgrade automatically. After the installation of the packages is complete, reboot the system to start the upgrade process.
+
+After the upgrade has started, the only way to access the system during the
+upgrade process is via ssh with a user called `migration`:
+
+[listing]
+tux > sudo ssh migration@IP_OF_INSTANCE
+
+[NOTE]
+There is no need to provide any other information or key. The known SSH keys on the system to be upgraded have been imported into the upgrade system. Password-based login is not possible.
+
+== Debugging the Migration
+In case of a upgrade failure, consult the migration log. The file `/etc/issue` will contain a pointer to the respective log file.
```

Comment:
@fsundermeyer I have left this ```migration``` because the name of the log file is 
```distro_migration.log```

Reply:
looks good to me